### PR TITLE
fixed up issue 251

### PR DESCRIPTION
**Pull request by Pomax**
made default view the same as 'normal' text size. Updated text size change to only occur if localStorage has a text size entry instead of also trying to trigger 'normal' in the absence of a localStorage value

### CODE REVIEW

**Comment by Pomax** [comment]
https://github.com/mozilla/webpagemaker/issues/251 can be closed on merge

File: css/jsbin-codemirror-theme.css [hunk 1]
```diff
@@ -28,13 +28,13 @@
   font-family: "Myriad Pro", "myriad-pro", Helvetica, Arial, sans-serif;
   color: #333;
   font-size: 14px;
-  line-height: 16px;
+  line-height: 15.75px;
```
Comment (brianloveswords) [inline css/jsbin-codemirror-theme.css:31]: WebKit doesn't support fractional pixels so this will round down to `15px`. That might be fine for these cases.

Comment (Pomax) [inline css/jsbin-codemirror-theme.css:31]: that would actually be fine - do we know if webkit is in error here, or is that sanctioned behaviour?

Comment (brianloveswords) [inline css/jsbin-codemirror-theme.css:31]: Not a webkit error, it just doesn't support that level of granularity so it just drops the decimal.

**Pull request merged**